test/test_helper.rb

```diff
@@ -1,9 +1,64 @@
 Bundler.require(:default, :test)
+
+require 'simplecov'
+SimpleCov.start
+
 require 'minitest/autorun'
 require 'webmock/minitest'
 require 'pp'
+require 'json_api_resource'
+require 'json_api_client'
 
-# test resources
-class User < JsonApiClient::Resource
+class UserClient < JsonApiClient::Resource
   self.site = "http://localhost:3000/api/1"
 end
+
+class UserResource < JsonApiResource::Resource
+  class << self
+    def client_klass
+      UserClient
+    end
+
+    def schema
+      { id: 0,
+        name: ""
+      }
+    end
```

Comment:
It's the existing paradigm. I would like to move away from it. Maybe we might as well now that I broke `property`